src/styles/index.css

```diff
@@ -0,0 +1,70 @@
+.stopwatch__circle,
+.stopwatch__second,
+.stopwatch__min,
+.stopwatch,
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@keyframes hand-rotate {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+body {
+  margin: 0;
+  height: 100vh;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  color: #000;
+  border: 1vmin;
+  border-style: dotted;
+  border-radius: 50%;
+}
+
+.stopwatch__min {
+  width: 3vmin;
+  height: 20vmin;
+  background-color: #0700ff;
+  animation: hand-rotate 3600s infinite steps(60);
+  bottom: 50%;
+  transform-origin: bottom;
+  position: absolute;
+}
```

Comment:
The `position: absolute;` property is duplicated for `.stopwatch__min` and `.stopwatch__second`. Consider creating a general utility class for absolute positioning and apply it to the necessary elements in your HTML. This will help to avoid code duplication and make your CSS more maintainable.